common.h

```diff
@@ -306,6 +307,13 @@ typedef int blasint;
 #define SIZE	8
 #define  BASE_SHIFT 3
 #define ZBASE_SHIFT 4
+#elif defined(BFLOAT16_ONLY)
```

Comment:
I do not think we need to introduce BFLOAT16_ONLY build flag.
The type of FLOAT is the only difference. Can we simplyl use XFLOAT instead of FLOAT for, for example, C matrix in kernel/arm64/bgemm_beta.c ?